Simplenote/NoteEditorViewController+Swift.swift

```diff
@@ -926,6 +926,18 @@ extension NoteEditorViewController {
             noteEditor.scrollToSelectedLocation()
         }
     }
+
+    @objc
+    func getRoundedFontSize() -> Int {
+        var currentFontSize = Options.shared.fontSize
+        let fontSizeIsNotValid = currentFontSize % SimplenoteConstants.NoteFontSizeStep != .zero
```

Comment:
Mind verifying if we **do** need this? it's possible that calling the rounding method, if it's already rounded, won't do anything.